jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -407,12 +407,10 @@ private String debugId() {
               computeDurationViaNanoTicks,
               tags,
               references);
-      if (context.isSampled()) {
-        metrics.spansSampled.inc(1);
-      } else {
-        metrics.spansNotSampled.inc(1);
+      if (!context.isSampled()) {
+        metrics.spansStartedNotSampled.inc(1);
       }
-      metrics.spansStarted.inc(1);
+      metrics.spansStartedSampled.inc(1);
```

Comment:
this should be on the `else` part

Reply:
is the test wrong?

Reply:
Good catch. There was no test for this case, so, I fixed this and added a test. Originally, the code had this:

```
      if (context.isSampled()) {
        metrics.spansSampled.inc(1);
      } else {
        metrics.spansNotSampled.inc(1);
      }
metrics.spansStarted.inc(1);
```

A previous iteration removed the `spansSampled` and kept the `spansStarted`, but looks like the correct is to have `spansSampled` and `spansNotSampled`, removing `spansStarted`.